content/en/docs/rules/supported-fields.md

```diff
@@ -84,19 +89,18 @@ fd.rip.name     Domain name associated with the remote IP address.
 fd.dev          device number (major/minor) containing the referenced file
 fd.dev.major    major device number containing the referenced file
 fd.dev.minor    minor device number containing the referenced file
-```
 
-## `proc` & `thread` Field Class
 
-Fields for running or spawned processes or threads.
+```
+
+### Field Class: process
```

Comment:
```suggestion
### Field Class: proc
```

Reply:
Perhaps we should fix that here:
https://github.com/falcosecurity/libs/blob/eecc959cc2ce7144d841c3f80d4a6271905a3709/userspace/libsinsp/filterchecks.cpp#L1861

?